plugin/cmd/kube-batchjob-controller/app/options/options.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import "github.com/spf13/pflag"
+
+type BatchJobControllerOptions struct {
+	ApiServer  string
+	KubeConfig string
+	Policy     string
+}
+
+func NewBatchJobControllerServer() *BatchJobControllerOptions {
+	return &BatchJobControllerOptions{}
+}
+
+func (bjc *BatchJobControllerOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&bjc.ApiServer, "api-server", "m", "localhost:8080", "The address of api-server.")
```

Comment:
If it's run as a separate introspective component you don't need this, see other example.